.github/ISSUE_TEMPLATE.md

```diff
@@ -1,34 +1,20 @@
-
 # Before you proceed
 
 - Read the [wiki](https://github.com/ckb-next/ckb-next/wiki).
-- Try the latest code by building the software manually.
 - Search in open and closed [Issues](https://github.com/ckb-next/ckb-next/issues).
+- If possible, try the latest code by building the software manually.
 - File unrelated issues separately.
 - Use [Markdown](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet) to format the message.
-- There are 3 types of issues: new device, feature, bug. Use only one. Delete redundant sections before sending.
-
+- There are 3 types of issues: new device, feature, bug.
+Use only one. Delete redundant sections before sending.
+If your issue doesn't fit in any of those categories, please use the *Something else* field at the bottom.
```

Comment:
With my very limited attention span didn't understand that this is regarding the issue template, thus I suggest:

This issue template has 3 main sections of which you should use only 1:
- [New Device Support Request](#new-device-support-request),
- [Feature Request](#feature-request), or
- [Bug Report](#bug-report).

If your issue doesn't fit in any of those categories, please use the [#something-else] section at the bottom.
Delete the redundant sections before sending to increase the chance of a maintainer reading it.

Reply:
Yeah, that's much better. I'll rewrite it to that.